content/get_start/UseCase_Workflows/revit.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "PRM for Revit"
+date: 2022-09-21T15:00:28-07:00
+weight: 251
+draft: false
+pre: "<b>- </b>"
+---
+
+{{<line_break>}}
+
+#### A. Use Case 1: Application of the PRM measure to a user-model created in Revit
+
+This use case includes translatiing a BIM model created in Revit to a BEM model and applying PRM measure to automatically generate a baseline and a proposed model based on the PRM rules. 
+
+Revit is a building information modeling software used for planning, designing, constructing and managing buildings and infrastructure. The following workflow is based on the capabilities present in Revit 2024 and is likely applicable to other versions as well. 
```

Comment:
Add a link to Revit 2024 https://help.autodesk.com/view/RVT/2024/ENU/?guid=GUID-C81929D7-02CB-4BF7-A637-9B98EC9EB38B

Reply:
Done!